Makefile

```diff
@@ -61,6 +61,17 @@ k8s-client-gen: applyconfiguration-gen
 		--trim-path-prefix 	"$(GO_MODULE)" \
 		--output-base    	"."
 
+.PHONY: wg-policy-client-gen
+wg-policy-client-gen: applyconfiguration-gen
```

Comment:
Should we include this in the generate/generate-all targets?

Reply:
We agreed to keep it as is for now. Another option is to disable Renovate for this dependency.